composer.json

```diff
@@ -31,9 +31,9 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
-    "oro/platform": "~4.2.0",
+    "oro/platform": "5.0.*",
     "gaufrette/aws-s3-adapter": "^0.4.0",
-    "knplabs/gaufrette": "0.9.0",
+    "knplabs/gaufrette": "0.10.0",
```

Comment:
@Zigr1, Should we up this higher? look `0.11.1` was released and PHP 8.0 support was technically only added in `0.11.0`

Reply:
Yeah, that's certainly meant to be 11. I believe it was a typo